pandas/core/series.py

```diff
@@ -204,6 +204,17 @@ def __init__(
         self, data=None, index=None, dtype=None, name=None, copy=False, fastpath=False
     ):
 
+        if (
+            isinstance(data, SingleBlockManager)
+            and index is None
+            and dtype is None
+            and copy is False
+        ):
+            # GH#33357 called with just the SingleBlockManager
+            NDFrame.__init__(self, data)
+            self.name = name
+            return
+
```

Comment:
are there any places where we are actually setting `fastpath=True` now ? we should deprecate this

Reply:
Series.take, DataFrame._box_col_values.  I generally agree it would be nice to be rid of

Reply:
And several more places as well. 
But that's an independent discussion I would say